pyproject.toml

```diff
@@ -144,9 +145,26 @@ only-include = [
 [tool.hatch.build.targets.wheel]
 packages = ["src/tmlt"]
 artifacts = [
-    "src/tmlt/core/ext/"
+    "src/tmlt/core/ext/__init__.py",
+    "src/tmlt/core/ext/lib/__init__.py",
+    # Only include the needed copy of each library -- the build process
+    # generates multiple symlinks to each library, which are converted into
+    # separate, identical files when building the wheel.
+    "src/tmlt/core/ext/lib/*.so.*.*.*",
+    "src/tmlt/core/ext/lib/*[0-9].dylib",
```

Comment:
Seeing you list out the init files, I wonder whether we should get more specific here. E.g., instead of one, somewhat confusing mac pattern, we could do:
```
"src/tmlt/core/ext/lib/libarb-*.dylib"
"src/tmlt/core/ext/lib/libflint-*.dylib"
"src/tmlt/core/ext/lib/libgmp.*.dylib"
"src/tmlt/core/ext/lib/libmpfr.*.dylib"
```

Thoughts? It should be relatively robust to library version changes, but won't pick up anything we didn't intend.